pkg/util/helper/binding.go

```diff
@@ -80,6 +80,22 @@ func GetBindingClusterNames(targetClusters []workv1alpha2.TargetCluster) []strin
 	return clusterNames
 }
 
+// GetUsedBindingClusterNames will get used clusterName list from bind clusters field
+func GetUsedBindingClusterNames(targetClusters []workv1alpha2.TargetCluster) []string {
```

Comment:
I guess it is not clear enough for the function intention.
I can understand the returned cluster which has assigned replicas on it, but why return the whole list if `usedClusterNames == 0`?
Consider abstract or rename it?

Reply:
Right. There is no need to return the whole list. I have deleted the logic.